app/controllers/admin/projects_controller.rb

```diff
@@ -99,7 +99,7 @@ def sort
   private
 
   def redirect_if_not_owner_or_admin(project)
-    unless current_user.owner?(project) || current_user.is_administrator?
+    unless current_user.owner?(project) || current_user.administrator?
```

Comment:
I know you only use this twice but maybe the logic could be moved into the current_user model?

Reply:
This has already been removed in the next pass.